filter/filter.go

```diff
@@ -22,18 +22,23 @@ import (
 )
 
 type model struct {
-	textinput      textinput.Model
-	viewport       *viewport.Model
-	choices        []string
-	matches        []fuzzy.Match
-	selected       int
-	indicator      string
-	height         int
-	aborted        bool
-	quitting       bool
-	matchStyle     lipgloss.Style
-	textStyle      lipgloss.Style
-	indicatorStyle lipgloss.Style
+	textinput          textinput.Model
+	viewport           *viewport.Model
+	choices            []string
+	matches            []fuzzy.Match
+	selected           int
```

Comment:
We should probably rename `selected` to `cursor` since I guess that better represents what state it is holding.

Reply:
I agree with that. I was a bit unsure about naming variables in general.